app/views/govuk_publishing_components/components/_big_number.html.erb

```diff
@@ -16,7 +16,19 @@
   end
 
 %>
-<% if number %>
+<% if number
+  # We amend the capture here if it ends with a "+" to account for small symbols awkwardly sitting above the baseline in our typeface and amend them to sit at the baseline for the purpose of visual harmony
```

Comment:
Could we abstract this into a helper, so we don't have long chunk of logic here? See [example for Intervention Banner](https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/intervention_helper.rb)